Source/m_cheat.c

```diff
@@ -457,6 +458,15 @@ char buf[3];
   entry = G_LookupMapinfo(epsd, map);
   if (!entry)
   {
+    char *next = MAPNAME(epsd, map);
+
+    if (W_CheckNumForName(next) == -1)
+    {
+      dprintf("IDCLEV target not found: %s", next);
+      return;
+    }
+
+#if 0
   // Catch invalid maps.
```

Comment:
I prefer to keep the original checks in place, although they are redundant now.

Reply:
But then idclev doesn't work for MAP > 32 etc. PrBoom+ alters these checks.

Reply:
My idea was to put the old "Catch invalid maps." check inside the `if (W_CheckNumForName(next) == -1)` braces instead of commenting it out. Sure, that'd be redundant, but we would still keep the original code around. Just as we did with the new `if (!entry)` code.